src/main/java/DilemmaDetector/Consequences/DecisionCostEvaluator.java

```diff
@@ -0,0 +1,73 @@
+package DilemmaDetector.Consequences;
+
+import project.Decision;
+import project.Living_entity;
+import project.MyFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
Nowa klasa, żeby nie trzeba było modyfikować DecisionCostCalculator

Reply:
A czemu nie? W końcu ona chyba do tego ma służyć. DecisionCostCalculator bez CustomPhilosophy to jak żołnierz bez karabinu 🔫

Reply:
Ok, to zrobiłem.